libandroid-navigation-ui/src/main/res/layout/feedback_bottom_sheet_layout.xml

```diff
@@ -6,13 +6,32 @@
     android:layout_height="wrap_content"
     android:background="@drawable/feedback_bottomsheet_background">
 
+    <TextView
+        android:id="@+id/reportFeedback"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="16dp"
+        android:paddingBottom="16dp"
+        android:paddingEnd="24dp"
```

Comment:
should we define all dimensions in separate xml?

Reply:
Already on it. This is still WIP

Reply:
btw, fyi this work could have been started as `draft` PR, then WIP in title is redundant.